[2024-02-02 08:09:57,909] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_28.dim_mysql manual__2024-02-02T06:43:07.744045+00:00 [queued]>
[2024-02-02 08:09:57,933] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_28.dim_mysql manual__2024-02-02T06:43:07.744045+00:00 [queued]>
[2024-02-02 08:09:57,934] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 08:09:57,935] {taskinstance.py:1357} INFO - Starting attempt 2 of 3
[2024-02-02 08:09:57,935] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 08:09:57,972] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): dim_mysql> on 2024-02-02 06:43:07.744045+00:00
[2024-02-02 08:09:57,985] {standard_task_runner.py:52} INFO - Started process 8412 to run task
[2024-02-02 08:09:57,989] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_28', 'dim_mysql', 'manual__2024-02-02T06:43:07.744045+00:00', '--job-id', '455', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpwpaf5ao3', '--error-file', '/tmp/tmpxr8zcsd4']
[2024-02-02 08:09:57,990] {standard_task_runner.py:80} INFO - Job 455: Subtask dim_mysql
[2024-02-02 08:09:58,208] {task_command.py:369} INFO - Running <TaskInstance: project_6_28.dim_mysql manual__2024-02-02T06:43:07.744045+00:00 [running]> on host 20775c919683
[2024-02-02 08:09:58,523] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_28
AIRFLOW_CTX_TASK_ID=dim_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-02-02T06:43:07.744045+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-02T06:43:07.744045+00:00
[2024-02-02 08:09:58,525] {mysql.py:82} INFO - Executing: CREATE TABLE IF NOT EXISTS case_table (
    id INT ,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
);


-- CREATE TABLE IF NOT EXISTS province_table (
--     province_id INT PRIMARY KEY,
--     province_name VARCHAR(255) NOT NULL
-- );

-- CREATE TABLE IF NOT EXISTS district_table (
--     district_id INT PRIMARY KEY,
--     province_id INT,
--     district_name VARCHAR(255) NOT NULL,
--     FOREIGN KEY (province_id) REFERENCES province_table(province_id)
-- );


CREATE TABLE IF NOT EXISTS province_table AS
select DISTINCT kode_prov, nama_prov  from raw_covid_data;

CREATE TABLE IF NOT EXISTS district_table AS
select distinct kode_kab, kode_prov , nama_kab from raw_covid_data;
[2024-02-02 08:09:58,585] {base.py:68} INFO - Using connection ID 'mysql_data_staging' for task execution.
[2024-02-02 08:09:58,696] {dbapi.py:208} INFO - Running statement: CREATE TABLE IF NOT EXISTS case_table (
    id INT ,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
);


-- CREATE TABLE IF NOT EXISTS province_table (
--     province_id INT PRIMARY KEY,
--     province_name VARCHAR(255) NOT NULL
-- );

-- CREATE TABLE IF NOT EXISTS district_table (
--     district_id INT PRIMARY KEY,
--     province_id INT,
--     district_name VARCHAR(255) NOT NULL,
--     FOREIGN KEY (province_id) REFERENCES province_table(province_id)
-- );


CREATE TABLE IF NOT EXISTS province_table AS
select DISTINCT kode_prov, nama_prov  from raw_covid_data;

CREATE TABLE IF NOT EXISTS district_table AS
select distinct kode_kab, kode_prov , nama_kab from raw_covid_data;, parameters: None
[2024-02-02 08:09:58,699] {dbapi.py:216} INFO - Rows affected: 0
[2024-02-02 08:09:58,784] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_28, task_id=dim_mysql, execution_date=20240202T064307, start_date=20240202T080957, end_date=20240202T080958
[2024-02-02 08:09:58,852] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 08:09:59,036] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
